Document cli options that can be set in configuration (#2693)
gohugoio/hugoDocs: content/en/getting-started/configuration.md

[kinds]: /getting-started/glossary/#page-kind

###### disableLanguages

> See [Disable a Language](/content-management/multilingual/#disable-a-language)

Reviewer: ```suggestion
See [disable a language](/content-management/multilingual/#disable-a-language).
```

(`bool`) Enable generation of `robots.txt` file. Default is `false`.

###### environment

> (`string`) Build environment. Default is _production_ when called as  `hugo` and _development_ when called as `hugo server`. See [Configuration directory](https://gohugo.io/getting-started/configuration/#configuration-directory)

Reviewer: ```suggestion
(`string`) Build environment. Default is `production` when running`hugo` and `development` when running `hugo server`. See [Configuration directory](https://gohugo.io/getting-started/configuration/#configuration-directory).
```

(`bool`) If true, auto-detect Chinese/Japanese/Korean Languages in the content. This will make `.Summary` and `.WordCount` behave correctly for CJK languages. Default is `false`.

###### ignoreCache

> (`bool`) Ignores the cache directory. Default is `false`.

Reviewer: ```suggestion
(`bool`) Ignore the cache directory. Default is `false`.
```

###### ignoreVendorPaths

> (`string`) ignores any _vendor for module paths matching the given Glob pattern. Default is `false`.

Reviewer: ```suggestion
(`string`) Ignore vendored modules that match the given [Glob] pattern within the `_vendor` directory .

[Glob]: https://github.com/gobwas/glob?tab=readme-ov-file#example]
```

See [Configure Languages](/content-management/multilingual/#configure-languages).

###### layoutDir

> (`string`) The directory where Hugo reads the layouts from. Default is `layouts`.

Reviewer: ```suggestion
(`string`) The directory that contains templates. Default is `layouts`.
```

See [configure pagination](/templates/pagination/#configuration).

###### panicOnWarning

> (`bool`) Panic on first WARNING log. Default is `false`.

Reviewer: ```suggestion
(`bool`) Whether to panic on first the WARNING log. Default is `false`.
```

Author: Suggestions:
1. Whether to panic on first WARNING. 
2. Whether to panic on the first WARNING.

Author: What about that last one? I would use:
>Whether to panic on first WARNING.

(`bool`) Whether to pluralize automatic list titles. Applicable to section pages. Default is `true`.

###### printI18nWarnings

> (`bool`) Print missing translations. Default is `false`.

Reviewer: ```suggestion
(`bool`) Whether to print missing translations to the console. Default is `false`.
```

Author: my suggestion - see global comment:
  (`bool`) Whether to log WARNINGs for each missing translation. Default is `false`.

###### printPathWarnings

> (`bool`) Print warnings on duplicate target paths etc. Default is `false`.

Reviewer: ```suggestion
(`bool`) Whether to print warnings to the console when Hugo publishes two or more files to the same path. Default is `false`.
```

Author: my suggestion - see global comment:
(`bool`)  Whether to log WARNINGs when Hugo publishes two or more files to the same path. Default is `false`.

###### printUnusedTemplates

> (`bool`) Print warnings on unused templates. Default is `false`.

Reviewer: ```suggestion
(`bool`) Whether to print a warning to the console for each unused template. Default is `false`.
```

Author: my suggestion - see global comment:
(`bool`)  Whether to log WARNINGs for each unused template. Default is `false`.

See [Configure Taxonomies](/content-management/taxonomies#configure-taxonomies).

###### templateMetrics

> (`bool`) Display metrics about template executions. Default is `false`.

Reviewer: ```suggestion
(`bool`) Whether to print template execution metrics to the console. Default is `false`.
```

See [Template metrics](/troubleshooting/performance/#template-metrics)

###### templateMetricsHints

> (`bool`) Calculate some improvement hints when _templateMetrics_ option `true`. Default is `false`. See [Template metrics](/troubleshooting/performance/#template-metrics)

Reviewer: ```suggestion
(`bool`) Whether to print template execution improvement hints to the console. Applicable when `templateMetrics` is `true`. Default is `false`. See [template metrics](/troubleshooting/performance/#template-metrics).
```
